packages/mock-addon/src/utils/array.js

```diff
@@ -1,8 +1,10 @@
 export function arrayEquals(a, b) {
-    return (
-        Array.isArray(a) &&
-        Array.isArray(b) &&
-        a.length === b.length &&
-        a.every((val, index) => val === b[index])
-    );
+    if (!Array.isArray(a) || !Array.isArray(b) || a.length !== b.length) {
+        return false;
+    }
+
+    const sortedA = a.sort();
+    const sortedB = b.sort();
```

Comment:
`sort` method mutates the array. we can keep the original array unchanged.

```
const sortedA = a.slice().sort(); // or [...a].sort()
```

Reply:
Good point!